docs/code-snippets/office-snippets.yaml

```diff
@@ -1490,13 +1490,23 @@ Office.DevicePermission#requestPermissions:member(1):
             .then((isGranted) => {
                 if (isGranted) {
                     console.log("Permission granted.");
+                    // Reload your add-in before you run code that uses the device capabilities.
+                    location.reload();
 
-                    // Do something when permission is granted.
+                    // Do something with the device capabilities.
```

Comment:
@samantharamon since location.reload is invoke, this line (code) never gets executed. maybe you can just remove this line as you already mentioned reload the addin.

Reply:
Removed comment here and from other applicable snippets.